hack/quick-install.sh

```diff
@@ -5,12 +5,29 @@ set -o xtrace
 
 export CERT_MANAGER_VERSION=v1.9.1
 export KNATIVE_EVENTING_VERSION=v1.14.3
+export JAEGER_OPERATOR_VERSION=v1.57.0
 
+# Production dependencies
 kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml
 kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-core.yaml
 kubectl rollout status deployment --namespace=cert-manager --timeout=30s
 kubectl rollout status deployment --namespace=knative-eventing --timeout=30s
 
+# Development dependencies
```

Comment:
Rather then just comment the code, add a log message so people running this script also know what is going on.

```suggestion
echo "Installing development dependencies"
```

Reply:
Sorry, I changed the approach. I was not convinced of adding it on the script, so I deleted the whole installation of the jaeger operator in favor of a deployment with the all-in-one image.